python/llm/src/bigdl/llm/ggml/convert_model.py

```diff
@@ -4,14 +4,14 @@
 import time
 
 
-def convert_model(model_path: str,
+def convert_model(input_path: str,
                   output_path: str = None,
                   model_family: str = 'llama',
                   dtype: str = 'q4_0'):
     """
     Convert Hugging Face llama-like / gpt-neox-like / bloom-like model to lower precision
 
-    :param model_path: str, path of model, for example `./llama-7b-hf`.
+    :param input_path: str, path of model, for example `./llama-7b-hf`.
```

Comment:
Need to document what is expected in the `input_path` folder, and how to obtain the folder for the user

Reply:
- I will add the document of input_path format in next pr, this pr  is being relied upon by other developers and should be merged as soon as possible. 
- I add an issue #8254  to track the document format problem.

Reply:
I will take this issue for input path format description